lib/rules/no-test-import-export.js

```diff
@@ -42,7 +45,7 @@ module.exports = {
       ImportDeclaration(node) {
         const importSource = node.source.value;
 
-        if (importSource.endsWith('-test')) {
+        if (importSource.endsWith('-test') && !isTestHelperImportSource(importSource)) {
```

Comment:
Hmm, I'm not 100% sure this is correct. We shouldn't be importing from `tests/helpers/something-test.js` (e.g. I think the first condition was fine, any file ending with `-test` is itself a test file not a helper file).

Reply:
I see why it was done. If you _literally_ write a module like `tests/helpers/some-verb-test.js` it would still error.